lib/versions.py

```diff
@@ -17,60 +14,17 @@ def _update_os_release():
             _os_release[key] = value
 
 
-class _RpmVerCmp:
-    # needs:
-    #   self.__bool__
-    #   self._release_separator
-    #   self.version
-    #   self.release
-
-    def compare(self, other):
-        if not isinstance(other, str):
-            other = str(other)
-        other_version, _, other_release = other.partition(self._release_separator)
-        theirs = (None, other_version, other_release if other_release else None)
-        # if 'theirs' is without release, omit it for 'ours' too
-        # to allow for things like <= '0.1.66'
-        ours = (None, self.version, self.release if other_release else None)
-        return rpm.labelCompare(ours, theirs)
-
-    def __lt__(self, other):
-        return bool(self) and self.compare(other) < 0
-
-    def __le__(self, other):
-        return bool(self) and self.compare(other) <= 0
-
-    def __eq__(self, other):
-        return bool(self) and self.compare(other) == 0
-
-    def __ne__(self, other):
-        return bool(self) and self.compare(other) != 0
-
-    def __ge__(self, other):
-        return bool(self) and self.compare(other) >= 0
-
-    def __gt__(self, other):
-        return bool(self) and self.compare(other) > 0
-
-    def __str__(self):
-        if self.release:
-            return f'{self.version}-{self.release}'
+class _Rhel:
+    def __init__(self, version=None):
+        if version is None:
+            _update_os_release()
+            v = _os_release['VERSION_ID'].split('.')
         else:
-            return self.version
-
-
-class _Rhel(_RpmVerCmp):
-    def __init__(self):
-        _update_os_release()
-        self._release_separator = '.'
-        v = _os_release['VERSION_ID'].split(self._release_separator)
+            v = version.split('.')
```

Comment:
I think `major, _, minor = version.partition('.')` would be safer here, because it would make `minor` into `None` on platforms where the version doesn't contain comma (compatibility of `lib` for Fedora would be nice, but not mandatory).